package.json

```diff
@@ -6,6 +6,7 @@
     "type": "git",
     "url": "https://github.com/tc39/tc39.github.io.git"
   },
+  "type": "module",
```

Comment:
it would have been better to make the files `.mjs` instead of using type module, ftr

Reply:
Hmm? Why?

Reply:
All type module does is change the parse goal of `.js`; it's not required for ESM, and using a `.mjs` extension may even be faster because the looking up the `type` field takes time.